website/docs/docs/get-started/develop-in-the-cloud.md

```diff
@@ -75,43 +75,33 @@ The Cloud IDE needs explicit action to save your changes. There are three ways y
-
-Review the following steps to set up your developer credentials:
-
-### Developer credentials
+In order to start experiencing the great features of the Cloud IDE, you need to first set up a [dbt Cloud development environment](/docs/collaborate/environments/dbt-cloud-environments). In the following steps, we outline how to set up developer credentials and access the IDE. If you're creating a new project, you will automatically configure this during the project setup. 
 
 The IDE uses developer credentials to connect to your data platform. These developer credentials should be specific to your user and they should *not* be super user credentials or the same credentials that you use for your production deployment of dbt.
 
-Follow the below steps to set up your developer credentials:
+Set up your developer credentials:
 
-1. Go to your [**Credentials**](https://cloud.getdbt.com/next/settings/profile#credentials) under the **Your Profile** settings.
+1. Go to your [**Credentials**](https://cloud.getdbt.com/settings/profile#credentials) under the **Your Profile** settings.
 2. Select the relevant project in the list.
 3. Click **Edit** on the bottom right of the page.
-4. Fill in the details in **Development Credentials**, including Dataset and Target Name. 
+4. Enter the details under **Development Credentials**.
 5. Click **Save.**
 
-Great job, you should now be able to access the Cloud IDE by clicking **Develop** on the navigation to start developing!
-
-<Lightbox src="/img/docs/dbt-cloud/refresh-ide/dev-credentials.png" width="100" height="100" title="Configure developer credentials in your Profile"/>
-
+<Lightbox src="/img/docs/dbt-cloud/refresh-ide/dev-credentials.jpg" width="100" height="100" title="Configure developer credentials in your Profile"/>
 
-### Access the Cloud IDE
 
-Now that you've set up your development environment and credentials, you should be able to access the Cloud IDE:
+6. Access the Cloud IDE by clicking **Develop** at the top of the page.
+7. Initialize your project and familiarize yourself with the IDE and its delightful [features](#cloud-ide-features).
 
-1. Log in with your dbt Cloud account. If you don't have one, [sign up](https://www.getdbt.com/signup/) for an account for free.
-2. Click **Develop** at the top of the page to take you to the IDE.
-3. Initialize your project and familiarize yourself with the IDE and its delightful [features](#cloud-ide-features).
-4. Nice job, you're ready to start developing and [building models](https://docs.getdbt.com/docs/get-started/getting-started/building-your-first-project/build-your-first-models) 🎉! 
+Nice job, you're ready to start developing and [building models](https://docs.getdbt.com/docs/get-started/getting-started/building-your-first-project/build-your-first-models) 🎉! 
```

Comment:
Make sure to use relative links so the fails when they are broken.
```suggestion
Nice job, you're ready to start developing and [building models](/docs/get-started/getting-started/building-your-first-project/build-your-first-models) 🎉! 
```

Reply:
I do think this flows better with the steps combined!

I wonder if it's not the best UX to send them into the middle of the Getting Started guide at this point? We have the next section on this page called [Build, compile and run...](https://deploy-preview-2834--docs-getdbt-com.netlify.app/docs/get-started/develop-in-the-cloud#build-compile-and-run-projects). Should we send them there? I wonder if this is pointing to a bigger problem of does the "Build, compile, run..." section belong on this page, or should it be a separate page (we don't need to solve this here - we could open an issue for this)? Also where did they come from to get here? Are they in the Getting started section? 

I know @nghi-ly is doing some work to tease apart the sample data/warehouse vs using your own data /warehouse, so maybe this will be a page we need OR this content can be moved from the "Build, compile, run..." section to the `getting-started/building-your-first-project/build-your-first-models` page? If you come up with any ideas or clear decisions, you can capture them in a new issue. If it's best left as-is, then that's fine too!

Reply:
gotcha! i first linked it to the 'build models' months ago because i had no idea they were linked to the guides and thought they were stand-alone pages 🙈 but since they're linked to the guides, I've removed the link as it doesn't make since the next section touches upon that. thank you @runleonarun !!

I've made those changes and think by removing that, it separates this doc from the guides, which allows the user to either use the ide for their own data or to follow the guides, if they want.